Examples/ShootingGameExample/Player.cs

```diff
@@ -0,0 +1,174 @@
+using Godot;
+using System;
+
+[MDAutoRegister]
+public class Player : KinematicBody2D
+{
+    public const string PLAYER_GROUP = "PLAYERS";
+
+    [Export]
+    public float MaxSpeed = 150f;
+    [Export]
+    public float Acceleration = 2000f;
+    [Export]
+    public float WeaponCooldown = 1f;
+
+    [MDBindNode("Camera2D")]
+    protected Camera2D Camera;
+
+    [MDBindNode("HitCounter")]
+    protected Label HitCounter;
+
+    protected bool IsLocalPlayer = false;
+
+    protected Vector2 MovementAxis = Vector2.Zero;
+    protected Vector2 Motion = Vector2.Zero;
+
+    protected float WeaponActiveCooldown = 0f;
+
+    protected PackedScene BulletScene = null;
+
+    protected int HitCounterValue = 0;
+  
+    [MDReplicated(MDReliability.Unreliable, MDReplicatedType.Interval)]
+    [MDReplicatedSetting(MDReplicator.Settings.GroupName, "PlayerPositions")]
+    [MDReplicatedSetting(MDReplicator.Settings.ProcessWhilePaused, false)]
+    [MDReplicatedSetting(MDClockedReplicatedMember.Settings.OnValueChangedEvent, nameof(OnPositionChanged))]
+    protected Vector2 NetworkedPosition;
+
+    [MDReplicated(MDReliability.Reliable, MDReplicatedType.OnChange)]
+    [MDReplicatedSetting(MDClockedReplicatedMember.Settings.OnValueChangedEvent, nameof(UpdateColor))]
+    protected Color NetworkedColor { get; set; }
+
+    // Called when the node enters the scene tree for the first time.
+    public override void _Ready()
+    {
+        AddToGroup(PLAYER_GROUP);
+        SetupPlayer(GetNetworkMaster());
+
+        //NetworkedShoot = new MDClockedNetworkValue<Vector2>(Vector2.Zero, IsLocalPlayer, ClockedPropertyMode.ONE_SHOT, MDReliability.Reliable);
```

Comment:
This is actually something I commented out because of not finishing OneShot replication, which we wanted to solve with MDRpc. I suppose I could remove it until I get that feature in and then add it back in. The example is currently broken because of lacking MDRpc that works with the game clock

Reply:
I removed this and added a todo comment instead